src/guillotine/fetchContent.ts

```diff
@@ -956,3 +983,84 @@ export const fetchContent: ContentFetcher = buildContentFetcher<AdapterConstants
     ...adapterConstants,
     componentRegistry: ComponentRegistry,
 });
+
+export function createResponse(props: FetchContentResult, context?: Context): GetStaticPropsResult<FetchContentResult> {
+    const {data, meta, error} = props;
+    const pageUrl = data?.get?.data?.target?.pageUrl;
+
+    if (meta.type === 'base:shortcut' && pageUrl) {
+        if (meta.renderMode !== RENDER_MODE.NEXT) {
+            // console.debug(`Returning 404 for shortcut in ${RENDER_MODE.NEXT} mode`);
+            // do not show shortcut targets in preview/edit mode
+            return {
+                notFound: true,
+            };
+        }
+
+        const destination = getUrl(pageUrl, meta, true);
+        // console.debug('Redirecting to', destination);
```

Comment:
```suggestion
```

Reply:
I left them on purpose for the future use ;)